docs/monitoring-aws-lambda.asciidoc

```diff
@@ -76,4 +76,4 @@ the next request until the extension has flushed all the data. This has a negati
 though it ensures that all APM data is sent to the APM server.
 
 === `ELASTIC_APM_LOG_LEVEL`
-The logging level to be used by both the APM Agent and the Lambda Extension. Supported values are `trace`, `debug`, `info`, `warning`, `error`, `fatal` and `panic`.
+The logging level to be used by both the APM Agent and the Lambda Extension. Supported values are `trace`, `debug`, `info`, `warning`, `error`, `critical` and `off` (the latter disables logging, except for fatal errors).
```

Comment:
> `off` (the latter disables logging, except for fatal errors).

I think it's a bit surprising that we would log when the level is "off", even for fatal level. Can you please explain the rationale behind this?

Reply:
I will implement a mechanism to remove logging when users specify `off` then (ie redirect logging output to /dev/null), as there is none built-in with Zap. 

The rationale behind the current implementation stems from:
- The lack of built-in mechanism (I wanted to strip down the implementation to a simple mapping to Zap levels)
- This [answer](https://github.com/uber-go/zap/issues/438) in the Zap repo

Reply:
Implemented in 508175e